website/components/top/Features.tsx

```diff
@@ -186,29 +186,29 @@ const cards = [
       name="Marp family: The official toolset"
       icon="https://icongr.am/octicons/package.svg?color=4a5568"
     >
-      Marp family has the rich toolset to assist your work.{' '}
+      The Marp ecosystem contains a rich toolset to assist your work.{' '}
       <a
         href="https://marketplace.visualstudio.com/items?itemName=marp-team.marp-vscode"
         rel="noopener noreferrer"
         target="_blank"
       >
-        <b>Marp for VS Code</b>
+        <b>The Marp for VS Code extension</b>
```

Comment:
We are treating "Marp for VS Code" as a proper noun to emphasize, so this change feels like to unnatural.